x-pack/platform/plugins/shared/fleet/common/types/models/epm.ts

```diff
@@ -126,6 +126,8 @@ export type InstallablePackage = RegistryPackage | ArchivePackage;
 
 export type AssetsMap = Map<string, Buffer | undefined>;
 
+export type PackagePolicyAssetsMap = AssetsMap & { __brand: 'PackagePolicyAssetsMap' };
```

Comment:
use type branding to make sure `AssetsMap` is not used as a `PackagePolicyAssetsMap` by error